src/array/cpu/spmm.h

```diff
@@ -141,7 +141,9 @@ void SpMMSumCsr(const BcastOff& bcast, const CSRMatrix& csr, NDArray ufeat,
   const bool no_libxsmm =
        bcast.use_bcast || std::is_same<DType, double>::value;
   if (!no_libxsmm) {
-    SpMMSumCsrLibxsmm<IdType, DType, Op>(bcast, csr, ufeat, efeat, out);
+    int ret = SpMMSumCsrLibxsmm<IdType, DType, Op>(bcast, csr, ufeat, efeat, out);
+    if (ret != 0)
+      SpMMSumCsrNaive<IdType, DType, Op>(bcast, csr, X, W, O);
```

Comment:
So it will try generating a CPU kernel every time even if it failed previously?  Need to benchmark how it impacts the performance (or we still need to provide a runtime switch that turns LibXSMM off).  I'm OK with the PR for now though.

Reply:
Yes, it will be good to have a runtime switch. Should we take it up later?

Reply:
Do you think it's easy to add it?  If so I would like to have it in this PR as well.